sphinx/remote-lsp.rst

```diff
@@ -1,25 +1,7 @@
 .. remote-lsp
 
-You can simulate a remote LSP with the following scripts:
-
-Start a server on any port (here I chose 8000):
-`$ nc -k -l localhost 8000 > test.nc`
-
-Then, use the following script to read on a file:
+You can simulate a remote LSP with the following script:
 ```
-touch $1
-inotifywait -m -e modify $1 | while read line
-do
-    if [ ! -s $1 ]; then
-       echo 'Waiting for content'
-    else
-	cat $1 | superbol lsp > superbol.log
-	> $1
-    fi
-done
+$ mkfifo /tmp/somewhere # possibly in a directory resulting from `mktemp -d`
+$ nc -l 8000 < /tmp/somewhere | superbol-free lsp > /tmp/somewhere
 ```
-
-and call it with `test.nc`.
```

Comment:
```suggestion
Once this is running, you can enter `tcp://localhost:8000` in the `superbol.lsp-path` configuration setting. SuperBOL Studio will then rely on this LSP server instead of a binary that is bundled in the extension.

Note: this "made up" remote server terminates once the first connection is closed.
```